hapi/build.gradle.kts

```diff
@@ -32,8 +32,7 @@ tasks.cloneHederaProtobufs {
     // uncomment below to use a specific tag
     //    tag = "v0.51.0"
     // uncomment below to use a specific branch
-    //     branch = "main"
-    branch = "add_admin_key"
+    branch = "main"
```

Comment:
This is already changed back to main. Please merge back latest develop.

Reply:
Done.